README.md

```diff
@@ -45,30 +45,59 @@ Append the following configuration
 The default value is used if the configuration was not provided
 The following variables can be set
 
-  * DB_NAME
+  * AUTH_DB_NAME
 	    	* database type. Current only postgres is supported
 	    	* default: postgres
 
-  * DB_USER
+  * AUTH_DB_USER
 	    	* The username used to access the database
 	    	* default: auth
 
-  * DB_PWD
+  * AUTH_DB_PWD
         * The password used to access the database
         * default: empty password
 
-  * DB_HOST
+  * AUTH_DB_HOST
         * The URL used to connect to the database
         * default: http://postgres
 
-  * KONG_URL
+  * AUTH_KONG_URL
         * The URL where the Kong service can be found
+        * If set to 'DISABLED' Auth won´t try to configure Kong and will generate secrets for the JWT tokens by itself.
         * default: http://Kong:8001
 
-  * TOKEN_EXP
+  * AUTH_TOKEN_EXP
         * Expiration time in second for generated JWT tokens
         * default: 420
 
+  * AUTH_TOKEN_CHECK_SIGN
+        * If Auth should verify received JWT signatures. Ative this will cause one query of overhead.
```

Comment:
**Whether** auth should verify received JWT signatures. **Enabling** this will cause one extra query to be performed.